Formula/neovim.rb

```diff
@@ -25,12 +12,6 @@ class Neovim < Formula
     sha256 "f563c067954a4c5f98ad2b01ee3219e5d1666f7000917474186127122cf5cc76" => :yosemite
   end
 
-  head do
-    url "https://github.com/neovim/neovim.git"
-
-    depends_on "luajit"
-  end
-
```

Comment:
Why remove support for --HEAD?

Reply:
The formula now unconditionally depends on `luajit`, so `head do` block is not needed anymore.
Also `head` support is still there. See line 6.